docs/users.rst

```diff
@@ -619,6 +619,7 @@ from_latest
 
 add_timestamp_to_release
   Boolean to control whether to append timestamp to explicitly specified release for autorebuilds.
+  Release value must be specified in the Dockerfile to enable this feature.
```

Comment:
add this as. note at the end of the description instead of in the middle like this.

Reply:
as you wish